runtime/util/cphelp.c

```diff
@@ -148,19 +148,23 @@ getModuleJRTURL(J9VMThread *currentThread, J9ClassLoader *classLoader, J9Module
 
 	if (NULL == jrtURL) {
 		if (J9_ARE_ALL_BITS_SET(javaVM->runtimeFlags, J9_RUNTIME_JAVA_BASE_MODULE_CREATED)) {
-			/* set jrt URL for the module */
-			if (NULL != module->moduleName) {
-				jrtURL = vmFuncs->copyStringToJ9UTF8WithMemAlloc(currentThread, module->moduleName, J9_STR_NONE, "jrt:/", 5, NULL, 0);
+			if (NULL == module->moduleName) {
+				goto _exit;
 			}
-
+			/* Set jrt URL for the module. */
+			const char *prependStr = "jrt:/";
+			const size_t prependStrLen = strlen(prependStr);
+			jrtURL = vmFuncs->copyJ9UTF8WithMemAlloc(
+					currentThread, module->moduleName, J9_STR_NONE, prependStr, prependStrLen, NULL, 0);
 			if (NULL == jrtURL) {
```

Comment:
In C files, variables need to be defined at the start of the scope. Otherwise, compilers (especially on Windows) will thrown a compilation error. Adding an `} else { `block should solve this issue.

Reply:
Right.